qemu/tests/blockdev_mirror_speed.py

```diff
@@ -46,13 +47,24 @@ def _set_valid_speed(jobid, speed):
             self.main_vm.monitor.cmd(
                 "block-job-set-speed", {'device': jobid, 'speed': speed})
 
+        def _check_valid_speed(jobid, speed):
+            job = job_utils.get_block_job_by_id(self.main_vm, jobid)
+            if job.get("speed") != speed:
+                self.test.fail("Speed:%s is not set as expected:%s"
+                               % (job.get("speed"), speed))
+            ck_speed = self.params.get_numeric("check_speed")
+            uspeed = self.params.get_numeric("ulimit_speed")
+            if speed > ck_speed or speed == uspeed:
+                self.check_block_jobs_running(
+                    self._jobs,
+                    self.params.get_numeric('mirror_running_timeout', 60)
+                    )
```

Comment:
We don't have to add this check, in fact, we have a check finally, that's self.wait_mirror_jobs_completed(), a non-running job should never complete.

Reply:
For small speed set, we just need to check job.speed =$set_speed, we don't need to check if the job is in running status, because for these small speed, the offset will never changed although the job is in running status.

In our original design, if the offset nerver changed, we will return False, and then will trigger: 'assert running, "Not all block jobs are running"'